packages/svelte/src/compiler/phases/3-transform/server/transform-server.js

```diff
@@ -411,6 +411,13 @@ const global_visitors = {
 			return b.literal(false);
 		}
 
+		if (rune === '$effect.root') {
+			const args = /** @type {import('estree').Expression[]} */ (
+				node.arguments.map((arg) => context.visit(arg))
+			);
+			return b.call('$.effect_root', ...args);
```

Comment:
AFAIK the `effect_root` function isn't exposed from the server entry point. Which makes me wonder why the test still passes.

Reply:
Needed to have a `html` field. Who knew.